spec/models/user_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before(:each) do
+    @user = User.new(Name: 'John Doe', Bio: 'Test Bio')
+  end
+
+  it 'is not valid if name absent' do
+    @user.Name = nil
+    expect(@user).to_not be_valid
+  end
+
+  it 'is not valid if photo absent' do
+    @user.photo = nil
+    expect(@user).to_not be_valid
+  end
+
+  it 'is not valid without bIO' do
+    @user.Bio = nil
+    expect(@user).to_not be_valid
```

Comment:
- [ ] Here as well. I would recommend adding the test for the positive scenarios as well. Ex:


```ruby
describe 'user name validations' do
  it { should validate_presence_of(:name) }
end
```

Reply:
This issue has been addressed well. Nice job! :+1: :+1: